modules/workergroup/locals.tf

```diff
@@ -0,0 +1,95 @@
+# Copyright 2022, Oracle Corporation and/or affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl
+
+locals {
+  tenancy_id = coalesce(var.tenancy_id, var.tenancy_ocid)
+}
+
+locals {
```

Comment:
Please make a single locals block. If there's a reason they should be in different locals block, then use a different file like we are doing in extensions:

- locals.tf
- templates.tf